src/components/select-field/SelectField.scss

```diff
@@ -18,7 +18,9 @@
 }
 
 .select-option-check-icon {
+    position: relative;
```

Comment:
Would `margin-left: -5px` work? It might allow you to avoid the position style.

Reply:
Yeah tried that 1st, didn't work because it moved the label along:

<img width="170" alt="Screen Shot 2019-10-21 at 5 52 55 PM" src="https://user-images.githubusercontent.com/1075325/67253285-bb05c780-f42b-11e9-9267-9284b038a1a0.png">

let me double check

Reply:
I can make that though will have to keep icon width to 20px